src/Extractor/ContentExtractor.php

```diff
@@ -607,16 +607,24 @@ function ($element, $currentEntity) {
                     continue;
                 }
 
-                $src = null;
+                $attributes = [];
                 foreach ($this->config['src_lazy_load_attributes'] as $attribute) {
                     if ($e->hasAttribute($attribute)) {
-                        $src = $e->getAttribute($attribute);
+                        $key = 'src';
+                        if ('data-srcset' === $attribute) {
+                            $key = 'srcset';
+                        }
+                        $attributes[$key] = $e->getAttribute($attribute);
                         $e->removeAttribute($attribute);
                     }
                 }
 
-                if (null !== $src) {
-                    $e->setAttribute('src', $src);
+                foreach (['src', 'srcset'] as $attr) {
+                    if (\is_array($attributes)
+                            && \array_key_exists($attr, $attributes)
+                            && null !== $attributes[$attr]) {
```

Comment:
I think `$attributes` is always an array at this point. Don't you think?

```suggestion
                    if (\array_key_exists($attr, $attributes) && null !== $attributes[$attr]) {
```

Reply:
Indeed, I'll push a fixup